### PR TITLE
Fix sample output

### PR DESCRIPTION
**Pull request by matthiasgubler**
Fix deprecated sample output documentation

## Proposed Changes 

- Use domain-mapping and domainmapping-webhook insted of the net-istio in sample output
- Use new sample output for knative eventing installation
-

### CODE REVIEW

File: docs/install/operator/knative-with-operators.md [hunk 1]
```diff
@@ -182,8 +182,8 @@ activator              1/1     1            1           18s
 autoscaler             1/1     1            1           18s
 autoscaler-hpa         1/1     1            1           14s
 controller             1/1     1            1           18s
-net-istio-webhook      1/1     1            1           12s
-net-istio-controller   1/1     1            1           12s
+domain-mapping         1/1     1            1           12s
+domainmapping-webhoop  1/1     1            1           12s
```
Comment (matthiasgubler) [inline docs/install/operator/knative-with-operators.md:186]:
```suggestion
domainmapping-webhook  1/1     1            1           12s
```

**Comment by nak3** [comment]
LGTM

Depending on the configuration, we can see `net-*` or some other deployments. But this is a __sample__ output so I think this change would be fine.

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Pull request merged**